LIPs/LIP-23.md

```diff
@@ -0,0 +1,24 @@
+---
+lip: 23
+title: Is Michael Jordan the GOAT ? 
+author: Nico Vergauwen (@kyriediculous)
+type: Informational
+status: Proposed
+created: 2020-05-05
+discussions-to: https://en.wikipedia.org/wiki/Michael_Jordan
+requires (*optional): <LIP number(s)>
```

Comment:
Can you remove the `requires`, `replaces`, and `part-of` fields since they're not applicable?

Reply:
done !

Reply:
haha just occurred to me why you assigned the number 23. Nice touch!